### PR TITLE
feat(TPG>=5.4)!: add vpc_network_sources in access level

### PR DESCRIPTION
**Pull request by imrannayer**

### CODE REVIEW

**Comment by imrannayer** [comment]
@g-awmalik plz review

File: examples/access_level_vpc_ip/main.tf [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+module "access_context_manager_policy" {
+  source  = "terraform-google-modules/vpc-service-controls/google"
+  version = "~> 5.0"
+
+  parent_id   = var.parent_id
+  policy_name = "int_test_vpc_sc_policy_${random_id.random_suffix.hex}"
+}
+
+# Create Network with a subnetwork and private service access for both netapp.servicenetworking.goog and servicenetworking.googleapis.com
+
```
Comment (g-awmalik) [inline examples/access_level_vpc_ip/main.tf:30]: nit: whitespace

File: examples/access_level_vpc_ip/main.tf [hunk 2]
```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2019 Google LLC
```
Comment (g-awmalik) [inline examples/access_level_vpc_ip/main.tf:2]:
```suggestion
 * Copyright 2024 Google LLC
```

Here and throughout.

File: examples/access_level_vpc_ip/variables.tf [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# variable "protected_project_id" {
+#   description = "Project number of the project INSIDE the regular service perimeter."
+#   type        = number
+# }
+
+# variable "ip_subnetworks" {
+#   description = "A list of CIDR block IP subnetwork specification. May be IPv4 or IPv6. Note that for a CIDR IP address block, the specified IP address portion must be properly truncated (i.e. all the host bits must be zero) or the input is considered malformed. For example, \"192.0.2.0/24\" is accepted but \"192.0.2.1/24\" is not. Similarly, for IPv6, \"2001:db8::/32\" is accepted whereas \"2001:db8::1/32\" is not. The originating IP of a request must be in one of the listed subnets in order for this Condition to be true. If empty, all IP addresses are allowed."
+#   type        = list(string)
+# }
```
Comment (g-awmalik) [inline examples/access_level_vpc_ip/variables.tf:35]: Comments? Are they needed?

**Review by g-awmalik — APPROVED** [review]

**Pull request merged**